aten/src/ATen/core/boxing/impl/make_boxed_from_unboxed_functor.h

```diff
@@ -171,9 +166,8 @@ namespace impl {
       guts::if_constexpr<guts::typelist::contains<supported_primitive_arg_types, T>::value>([] {
         /* everything is ok, this is a primitive type */
       }, /* else */ [] {
-        // TODO This is called for each operator call and potentially expensive.
-        // This check should be moved to operator registration time instead.
-        TORCH_CHECK(c10::isCustomClassRegistered<T>(), "Tried to use undefined class ", c10::util::get_fully_qualified_type_name<T>(), " as output");
+        /* otherwise this must be an instance of a valid custom class, since it can only
```

Comment:
This comment is confusing because output types don't come from IValue, you don't have an IValue yet when calling into here. What you probably mean is something like "if this was invalid, it would fail later on when creating an IValue from it"